src/ledger/NetworkConfig.cpp

```diff
@@ -1419,12 +1423,14 @@ SorobanNetworkConfig::createCostTypesForV22(AbstractLedgerTxn& ltx,
 }
 
 void
-SorobanNetworkConfig::createCostTypesForV24(AbstractLedgerTxn& ltx,
+SorobanNetworkConfig::createCostTypesForV25(AbstractLedgerTxn& ltx,
```

Comment:
should this method exist at all in non-vNext?

Reply:
I don't mind it - we'll need a runtime check for this being p25 anyways. I generally prefer to have less ifdef guards - the code is simpler and this ensures that we're actually using the runtime checks from the get-go.